generators/heroku/templates/application-heroku.yml.ejs

```diff
@@ -74,3 +74,15 @@ spring:
             uri: ${BONSAI_URL}
   <%_ } _%>
 <%_ } _%>
+<%_ if (useOkta) { _%>
+    security:
+        oauth2:
+            client:
+                provider:
+                    oidc:
+                        issuer-uri: ${OKTA_OAUTH2_ISSUER}
+                registration:
+                    oidc:
+                        client-id: ${OKTA_OAUTH2_CLIENT_ID_WEB}
+                        client-secret: ${OKTA_OAUTH2_CLIENT_SECRET_WEB}
```

Comment:
This is different from the values in the documentation. The documentation mentions only `OKTA_OAUTH2_CLIENT_ID` resp. `OKTA_OAUTH2_CLIENT_SECRET` but they don't exist. the `_WEB` variables and work just fine.